tests/installation/first_boot.pm

```diff
@@ -31,7 +31,7 @@ sub run {
         select_console 'root-ssh';
         return;
     }
-    elsif (check_var('DESKTOP', 'textmode') || get_var('BOOT_TO_SNAPSHOT')) {
+    elsif (check_var('DESKTOP', 'textmode') || get_var('BOOT_TO_SNAPSHOT') || check_var('DESKTOP', 'serverro')) {
```

Comment:
better sort the two desktop checks together, e.g. `check_var('DESKTOP', 'textmode') || check_var('DESKTOP', 'serverro')`, but even more common what we have in code is `get_var('DESKTOP', '') =~ /textmode|serverro/`